README.md

```diff
@@ -1 +1,225 @@
+author and/or relevant stakeholders to discuss the issues in greater detail. In
+either case a summary from the meeting will be posted back to the RFC pull
+request.
+
+Maintainers of related repositories makes final decisions about RFCs after the
+benefits and drawbacks are well understood. These decisions can be made at any
+time, but the maintainers will regularly issue decisions. When a decision is made,
+the RFC pull request will either be merged or closed. In either case, if the
+reasoning is not clear from the discussion in thread, the maintainer of RFC
+repository will add a comment describing the rationale for the decision.
+
+## Implementing an RFC
+[Implementing an RFC]: #implementing-an-rfc
+
+Some accepted RFCs represent vital features that need to be implemented right
+away. Other accepted RFCs can represent features that can wait until some
+arbitrary developer feels like doing the work. Every accepted RFC has an
+associated issue tracking its implementation in the Exonum repository; thus that
+associated issue can be assigned a priority via the triage process that the
+team uses for all issues in the Exonum repository.
+
+The author of an RFC is not obligated to implement it. Of course, the RFC
+author (like any other developer) is welcome to post an implementation for
+review after the RFC has been accepted.
+
+If you are interested in working on the implementation for an "active" RFC, but
+cannot determine if someone else is already working on it, feel free to ask
+(e.g. by leaving a comment on the associated issue).
+
+## RFC Postponement
+[RFC Postponement]: #rfc-postponement
+
+Some RFC pull requests are tagged with the "postponed" label when they are
+closed (as part of the rejection process). An RFC closed with "postponed" is
+marked as such because we want neither to think about evaluating the proposal
+nor about implementing the described feature until some time in the future, and
+we believe that we can afford to wait until then to do so. Historically,
+"postponed" was used to postpone features until after 1.0. Postponed pull
+requests may be re-opened when the time is right.
+
+Usually an RFC pull request marked as "postponed" has already passed an
+informal first round of evaluation, namely the round of "do we think we would
+ever possibly consider making this change, as outlined in the RFC pull request,
+or some semi-obvious variation of it." (When the answer to the latter question
+is "no", then the appropriate response is to close the RFC, not postpone it.)
+
+## Help this is all too informal!
+[Help this is all too informal!]: #help-this-is-all-too-informal
+
+The process is intended to be as lightweight as reasonable for the present
+circumstances. As usual, we are trying to let the process be driven by
+consensus and community norms, not impose more structure than necessary.
+
+## License
+
+The Exonum RFCs are licensed under the Creative Commons Non-Commercial
+Share-Alike International License (version 4.0).
```

Comment:
Are you sure this is a good choice? Compare with [BIP licensing](https://github.com/bitcoin/bips/blob/master/bip-0002.mediawiki#bip-licensing).